e2e-pw/src/oss/fixtures/dedent.ts

```diff
@@ -0,0 +1,22 @@
+import dedent from "ts-dedent";
+
+export function dedentPythonCode(code: string): string {
```

Comment:
curious why you need this?

Reply:
because `ts-dedent` is not handling some edge cases :/